package.json

```diff
@@ -3,43 +3,68 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "node main.js",
-    "lint": "eslint ./routes ./service ./util ./browser",
+    "start": "ts-node main.js",
+    "start-chat": "webpack-dev-server --mode development --open --hot",
+    "lint": "eslint ./routes ./service ./util ./browser ./react",
     "test": "nyc ./tests/everything.sh",
     "update-brassau": "browserify -t babelify ./browser/brassau.js -o public/javascripts/brassau.js",
     "prepare": "(cd node_modules/thingtalk; yarn); (cd node_modules/thingengine-core; yarn; rm -fr node_modules); yarn update-brassau",
-    "coverage": "nyc report --reporter=text-lcov | coveralls"
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
+    "build": "webpack --mode production",
```

Comment:
Build should be part of "prepare" right? Otherwise it won't run by default.

Reply:
Ok